package.json

```diff
@@ -2,7 +2,20 @@
   "name": "@railgun-reloaded/subsquid-client",
   "version": "1.0.0",
   "description": "TypeScript client for interacting with Railgun Subsquid GraphQL APIs with enhanced type safety and filtering capabilities.",
+  "homepage": "https://github.com/railgun-reloaded/subsquid-client#readme",
+  "bugs": {
+    "url": "https://github.com/railgun-reloaded/subsquid-client/issues"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+ssh://git@github.com/railgun-reloaded/subsquid-client.git"
+  },
+  "license": "MIT",
+  "author": "Railgun Project Contributors",
   "main": "src/client.js",
+  "directories": {
```

Comment:
Should we add the `src` directory here ?

Reply:
No, you don’t need to add "src" to the "directories" field in package.json. The "directories" field is primarily used for metadata in some package managers and tools, like npm init, to specify the location of directories like bin, lib, or test.

Since "directories.test" is already set to "test", it just indicates where your test files are located. Adding "src" wouldn’t provide any functional benefit unless a tool you’re using specifically requires it.

Reply:
Maybe @emilbayes can join the conversation 👀